src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/advanced/GregtechMetaTileEntity_Adv_EBF.java

```diff
@@ -268,7 +271,9 @@ public void onPostTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
                     if (!this.depleteInputFromRestrictedHatches(this.mPyrotheumHatches, 5)) {
                         if (mGraceTimer-- == 0) {
                             this.causeMaintenanceIssue();
-                            this.stopMachine();
+                            this.stopMachine(
+                                    ShutDownReasonRegistry.outOfFluid(
+                                            Objects.requireNonNull(FluidUtils.getFluidStack("pyrotheum", 10))));
```

Comment:
It's probably copy-paste error. Machine requires 5 (and not 10) of pyrotheum (as stated few lines above).

Reply:
Tooltips say it consumes 10L/s pyrotheum but the true working mechanism is to consume 5L/0.5s. And I choose to follow the tooltips